spec/defines/schemes/scheme_spec.rb

```diff
@@ -90,6 +90,7 @@
           'command' => '& C:\Windows\System32\powercfg.exe /create test'
         )
       end
+      it { should compile }
```

Comment:
These should all be:

```ruby
it { is_expected.to compile }
```

Reply:
Was going off the example I saw earlier, but figured something was off.  Fixed!